packages/babel-compat-data/scripts/build-data.js

```diff
@@ -306,7 +306,7 @@ const generateData = (environments, features) => {
         "The newly generated plugin/built-in data does not match the current " +
           "files. Re-run `make build-compat-data`."
       );
-      process.exit(1);
```

Comment:
Can you revert this commit? I realize that this function is in a `forEach` callback.